std/datetime.d

```diff
@@ -390,9 +390,14 @@ public:
             immutable result = FILETIMEToStdTime(&fileTime);
             static if(clockType == ClockType.second)
             {
-                // This should probably just use core.stdc.time.time, but dmc's
-                // time function seems to apply DST to time_t, which is
-                // incorrect and thus only works with dmc's other C functions.
+                // Ideally, this would use core.std.time.time, but the C runtime
```

Comment:
That's 2 walls of text for a minor comment on disabled test. Just file an issue and link it once?